openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/SCSoftmxTestUtil.java

```diff
@@ -137,7 +137,7 @@ public static LoadTestProcessDefinition getMiniMixLoadTestOptions(LoadTestProces
 			.addProjectToClasspath("openjdk.test.classloading")  // For sharedclasses-mix inventory
 			.addProjectToClasspath("openjdk.test.nio")           // For sharedclasses-mix inventory
 			.addProjectToClasspath("openj9.test.daa")         // For sharedclasses-mix inventory
-		   	.generateCoreDumpAtFirstLoadTestFailure(false)
+		   	.generateCoreDumpAtFirstLoadTestFailure(true)
```

Comment:
Why is this one changed from false to true?

Reply:
That was mistake. Fixed.